kview/src/main/java/dev/ikm/komet/kview/controls/skin/KLConceptNavigatorTreeViewSkin.java

```diff
@@ -207,6 +217,20 @@ public KLConceptNavigatorTreeViewSkin(KLConceptNavigatorControl treeView) {
         });
         treeView.setOnDragDone(_ -> resetSelection());
 
+        ScrollBar verticalBar = (ScrollBar) treeView.lookup(".scroll-bar:vertical");
+        verticalBar.skinProperty().addListener(new InvalidationListener() {
+            @Override
+            public void invalidated(Observable observable) {
+                if (verticalBar.getSkin() != null) {
+                    StackPane thumb = (StackPane) verticalBar.lookup(".thumb");
+                    thumb.addEventHandler(MouseEvent.MOUSE_PRESSED, _ -> isScrollBarDragging = false);
+                    thumb.addEventHandler(MouseEvent.MOUSE_DRAGGED, _ -> isScrollBarDragging = true);
+                    thumb.addEventHandler(MouseEvent.MOUSE_RELEASED, _ -> isScrollBarDragging = false);
+                    verticalBar.skinProperty().removeListener(this);
+                }
+            }
+        });
```

Comment:
Wow, really impressed by this ability to dynamically grab the thumb and add event handlers.